erpnext/manufacturing/doctype/bom/bom.py

```
@@ -1464,13 +1467,16 @@ def add_non_stock_items_cost(stock_entry, work_order, expense_account):
 
 
 def add_operating_cost_component_wise(
-	stock_entry, work_order=None, operating_cost_per_unit=None, op_expense_account=None
+	stock_entry, work_order=None, operating_cost_per_unit=None, op_expense_account=None, job_card=None
```

Ruff (0.12.2)

Unused function argument: `operating_cost_per_unit`

(ARG001)